src/test/java/roomescape/controller/AdminControllerTest.java

```diff
@@ -1,39 +1,184 @@
 package roomescape.controller;
 
 import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
 import org.springframework.test.annotation.DirtiesContext;
 
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.is;
+
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
 @DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
 public class AdminControllerTest {
-    
+
+    @BeforeEach
+    void setUp() {
+        Map<String, Object> adminParams = new HashMap<>();
+        adminParams.put("name", "어드민");
+        adminParams.put("email", "admin@gmail.com");
+        adminParams.put("password", "1234");
```

Comment:
```suggestion
        SignupRequest request = new SignupRequest("어드민", "admin@gmail.com", "1234");
```

Map이 아닌 Request Dto를 사용하면 더 간단하게 요청 객체를 만들 수 있습니다 :)